### DISCUSSION TITLE
neovim 有自动修复功能么？

### DISCUSSION THREAD

**Discussion by Linya-IronMan** [nshen/learn-neovim-lua#62] [Q&A]
类比VSCode 中的一样，当你使用了一个另一个文件 export 的变量的时候，可以通过快捷键让VSCode 自动将这个变量的引入添加到当前文件中。
比如 “import { add } from 'math'”. 这样子会被IDE自动添加
neovim不知道有没有这个功能

**Comment by nshen** [comment]
如果是按照我的配置的话，可以尝试一下 `gi` 快捷键，会尝试引入使用到的变量
https://github.com/nshen/learn-neovim-lua/blob/7af70083eaf469fccb2ad601eaeabac150d080e5/lua/keybindings.lua#L265-L270
